### PR TITLE
LPS-101814

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by lawrence-lee** [comment]
ci:test:project-templates

File: modules/sdk/project-templates/project-templates-theme/build.gradle [hunk 1]
```diff
@@ -1,5 +1,5 @@
 dependencies {
-	compileOnly group: "com.liferay", name: "com.liferay.project.templates", version: "4.5.2"
+	compileOnly group: "com.liferay", name: "com.liferay.project.templates", version: "4.5.3"
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-theme/build.gradle:2]: hey @lawrence-lee  this version doesn't exist yet.  Brian wont be able to publish the archetypes first before publishing `com.liferay.project.templates`

Comment (gamerson) [reply modules/sdk/project-templates/project-templates-theme/build.gradle:2]: can you remove these? Brian should be able to bump this.

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/hhuijser/liferay-portal/pull/3764. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**